src/www/status_interfaces.php

```diff
@@ -275,12 +275,28 @@
                   endif;
                 if ($ifinfo['status'] != "down"):
                   if ($ifinfo['dhcplink'] != "down" && $ifinfo['pppoelink'] != "down" && $ifinfo['pptplink'] != "down"):
-                    if ($ifinfo['ipaddr']):?>
+                    if ($ifinfo['ipaddr']):
```

Comment:
should split this up as well, not sure if this is the best approach to add conditionals where none existed before